src/components/HomePage.js

```diff
@@ -83,16 +126,23 @@ function HomePage() {
         return null;
     }
   };
-
   const renderModalContent = () => {
     switch (modalContent) {
       case "Add Job":
         return (
           <div>
             <div>
               <label>Name for New Job:</label>
-              <input type="text" className="form-control" />
+              <input
+                type="text"
+                className="form-control"
+                value={newJobName}
+                onChange={(e) => setNewJobName(e.target.value)}
+              />
             </div>
+            <Button variant="primary" onClick={handleAddJob}>
```

Comment:
Here, we don't need to add separate OK button, we can use the common OK and Cancel buttons available in Modal footer.

Reply:
In the modal footer we can do changes like this to make this functionality work 
<Modal show={showModal} onHide={handleCloseModal}>
      <Modal.Header closeButton>
        <Modal.Title>{modalContent} Confirmation</Modal.Title>
      </Modal.Header>
      <Modal.Body>{renderModalContent()}</Modal.Body>
      <Modal.Footer>
        <Button
          variant="primary"
          onClick={() => {
            if (modalContent === "Add Job") {
              handleAddJob(); // Trigger handleAddJob on clicking OK
            } else if (modalContent === "Remove Job") {
              handleDeleteJob(); // Trigger handleDeleteJob on clicking OK
            }
            handleCloseModal(); // Close the modal
          }}
        >
          OK
        </Button>
        <Button variant="secondary" onClick={handleCloseModal}>
          Cancel
        </Button>
      </Modal.Footer>
    </Modal>

Reply:
Resolved the Issue

Reply:
Sure, thanks!
Everything else LGTM.